benches/README.md

```diff
@@ -44,6 +44,30 @@ Notes:
 - On Graviton 3 and 4, RPO256 and RPX256 are run with SVE acceleration enabled.
 - On AMD EPYC 9R14, RPO256 and RPX256 are run with AVX2 acceleration enabled.
 
+### Data Structure Benchmarks
+We build cryptographic data structures incorporating these hash functions.
+What follows are benchmarks of operations on sparse Merkle trees (SMTs) which use the above `RPO_256` hash function.
+We perform a batched modification of 1,000 values in a tree with 1,000,000 leaves (with the `smt_hashmaps` feature to use the `hashbrown` crate).
+
+Times are given as a per-operation average.
+
+#### Scenario 1: SMT Batched Insertion
+
+| Function          | Sequential | Concurrent |
+| ----------------- | ---------- | ---------- |
+| AMD Ryzen 9 7950X | 205 µs     |  19 µs     |
+| Apple M1 Air      | 785 µs     | 450 µs     |
```

Comment:
Could we also add benchmarks I posted here: https://github.com/0xPolygonMiden/crypto/pull/365#issuecomment-2641822845? I had them for the whole operation - so, maybe doing for the entire batch (rather than per operation) is fine?

I would also add a column for the improvement of concurrent over sequential.